README.md

```diff
@@ -138,7 +138,7 @@ When type is `raid` specifies RAID chunk size as a string, e.g.: '512 KiB'.
 Chunk size has to be multiple of 4 KiB.
 
 ##### `encryption`
-This specifies whether or not the volume will be encrypted using LUKS.
+This specifies whether the volume will be encrypted using LUKS.
```

Comment:
I would prefer "This specifies that the pool . . ." - I prefer indicative rather than conditional or subjunctive

Reply:
Does "that ... will be encrypted" really cover the `false` value?

Reply:
hmm - probably not - so I guess it is ok as you have it